proposed/extended-coding-style-guide.md

```diff
@@ -247,6 +247,89 @@ class ClassName extends ParentClass implements
 }
 ```
 
+### Using traits
+
+The `use` keyword used inside the classes to implement traits MUST be
+declared one line after the classe's opening brace, and MUST be indented
+with 4 space.
+
+```
+<?php
+namespace Vendor\Package;
+
+use Vendor\Package\FirstTrait;
+
+class ClassName
+{
+    use FirstTrait;
+}
+```
+
+When the `use` declaration exceeded 120 lines, use muiltiple lines for
+implement the traits.
+
+```
+<?php
+namespace Vendor\Package;
+
+use Vendor\Package\FirstTrait;
+use Vendor\Package\SecondTrait;
+use Vendor\Package\ThirdTrait;
+use Vendor\Package\FourthTrait;
+use Vendor\Package\FivethTrait;
+use Vendor\Package\SixthTrait;
+use Vendor\Package\SeventhTrait;
+use Vendor\Package\EighthTrait;
+use Vendor\Package\NinethTrait;
+use Vendor\Package\TenthTrait;
+
+class ClassName
+{
+    use FirstTrait,
+        SecondTrait,
+        ThirdTrait,
+        FourthTrait,
+        FivethTrait,
+        SixthTrait,
+        SeventhTrait,
+        EighthTrait,
+        NinethTrait,
+        TenthTrait;
+}
+```
+
+When the class has nothing after the `use` declaration, the class
+closing brace MUST be one line after the `use` declaration.
+
+```
+<?php
+namespace Vendor\Package;
+
+use Vendor\Package\FirstTrait;
+
+class ClassName
+{
+    use FirstTrait;
+}
+```
+
+When the class has things after the `use` declaration, you MUST skip
```

Comment:
Let's not use the word 'things' and change skip one line to have a blank line after. See earlier in the spec (with namespaces) for correct and consistent language to use.